app/views/courses/_create_new_course.html.erb

```diff
@@ -1,20 +1,35 @@
-<h3>Create New Course</h3>
+<% @title = "Create New Course" %>
+
+<h4>Create New Course</h4>
 
 <%= form_for :newCourse, url: courses_path, builder: FormBuilderWithDateTimeInput do |f| %>
-  <%= f.error_messages %>
+  <%#= f.error_messages %>
 
-  <%= f.text_field :name, required: true,
+  <%= f.text_field :name, size: 3, required: true,
                           help_text: "The course ID used in URLs and DB keys. This field must be unique
-  and URL-able. Typically includes the course number and semester. Examples:
-  15213-f12, 15110-s11" %>
+      and URL-able. Typically includes the course number and semester. Examples:
+      15213-f12, 15110-s11",
+                          error_text: if @newCourse.errors.full_messages_for(:name).empty?
+                                        ""
+                                      else
+                                        @newCourse.errors.full_messages_for(:name)[0]
+                                      end %>
 
-  <%= f.text_field :semester, size: 3,
-                              help_text: "The current semester. Examples: f12, s13, m13" %>
+  <%= f.text_field :semester, size: 3, required: true,
+                              help_text: "The current semester. Examples: f12, s13, m13",
+                              error_text: if @newCourse.errors.full_messages_for(:semester).empty?
```

Comment:
It doesn't seem like there is a case where there will be errors in semester so not sure if this is necessary, correct me if I'm wrong though